src/index/opensearch.rs

```diff
@@ -14,7 +14,9 @@ use crate::IndexId;
 use crate::Limit;
 use crate::PrimaryKey;
 use crate::SpaceType;
+use crate::index::actor::AnnError;
 use crate::index::actor::Index;
+use crate::index::validator::validate_embedding_dimensions;
```

Comment:
In Rust convention is to import modules, not functions and call functions using module name: `validator::validate_embedding_dimensions`. Same for importing in `index::usearch`.
```suggestion
use crate::index::validator;
```